exercises/practice/meetup/.meta/example.jl

```diff
@@ -0,0 +1,3 @@
+function meetup(year, month, week, dayofweek)
+
+end
```

Comment:
This should become [exercises/practice/meetup/meetup.jl](https://github.com/exercism/julia/pull/764/files#diff-9e5779cec2b85194336bc974cd9f95cee7533d462fb88a6d6db7cd944c58f68a) and [exercises/practice/meetup/meetup.jl](https://github.com/exercism/julia/pull/764/files#diff-9e5779cec2b85194336bc974cd9f95cee7533d462fb88a6d6db7cd944c58f68a) should become this file.

How it works is:
- in the CI it will take the `example.jl` file and rename it to `meetup.jl`, and then run the tests on it
- in the web editor / CLI, it will download `meetup.jl` as the starting point for the student.

Switch the two files around and you should see the CI pass once again!